src/routes/index.js

```diff
@@ -173,3 +193,25 @@ function addCoreRoutes(app, router, middleware) {
 	app.use(controllers.errors.handleURIErrors);
 	app.use(controllers.errors.handleErrors);
 }
+
+async function addRemountableRoutes(app, router, middleware, mounts) {
+	await Promise.all(Object.keys(mounts).map(async (mount) => {
```

Comment:
doesnt look like `addRemountableRoutes` needs any async/await

Reply:
Good catch, initially the plugin hook was called within. I'll remove.